velocity/pom.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This shouldn't have been removed

Reply:
Sorry, that was an accident

Reply:
Fixed in https://github.com/GeyserMC/Floodgate/pull/18/commits/e8d4e18cc87c4e699543aa967a36b17ac1306efb